src/components/modal/z-modal/index.tsx

```diff
@@ -39,6 +39,10 @@ export class ZModal {
   @Prop()
   closable?: boolean = true;
 
+  /** if true, the modal content is scrollable */
+  @Prop()
+  scrollable?: boolean = true;
```

Comment:
Metterei comunque un nome della prop più parlante, perchè scrollare lo fa in entrambi i casi: **scrollInside**?